categories.md

```diff
@@ -0,0 +1,27 @@
+---   
+layout: post
+title: "Categories"
+permalink: /categories/
+---
+
+<div id="archives">
+ <ul>
+{% for category in site.categories %}
+  <div class="archive-group">
+    {% capture category_name %}{{ category | first }}{% endcapture %}
+    <div id="#{{ category_name | slugize }}"></div>
```

Comment:
What does the `slugize` filter do? I can't seem to find any mention of it anywhere 🤔